src/main/java/mate/academy/service/OrderServiceImpl.java

```diff
@@ -24,6 +29,7 @@ private List<Product> getAllProductsFromShoppingCart(Long userId) {
         Product macBook = new Product("MacBook Air 2020", BigDecimal.valueOf(1399));
         Product xiaomi = new Product("Xiaomi 12", BigDecimal.valueOf(499));
         List<Product> products = List.of(iphone, macBook, xiaomi);
+        logger.info("Successfully fetched {} products for user ID {}", products.size(), userId);
```

Comment:
The TODO comment on line 33 suggests adding a log message about successfully fetched data from DB. It seems you have already added an appropriate log message here, so you can remove the TODO comment.